gym/utils/renderer.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Callable, List, Optional
+
+NO_RETURNS_RENDER = ["human", None]
+
+
+class Renderer:
```

Comment:
This requires a bunch of comments describing what exactly it does, since it seems to have a pretty important role in the whole rendering setup